Source/WebCore/page/Quirks.cpp

```diff
@@ -1879,5 +1879,13 @@ bool Quirks::needsLaxSameSiteCookieQuirk() const
 
     return *m_needsLaxSameSiteCookieQuirk;
 }
+#if ENABLE(TEXT_AUTOSIZING)
```

Comment:
Could you add the radar and/or bugs.webkit.org?
See elsewhere in the code for example.